### PR TITLE
Adding Base template and about template

### LINKED ISSUE #76: Add base template

### PR DESCRIPTION
**Pull request by OmerCS8**
- Adding a base template with the main menu that connects to all views.
- Adding template fo the 'about' page
- Adding CSS for base and about
- The base template adjusts itself according to the logged in user, and displays guest mode if no one is logged in.
fix #76
Depends on #72

### CODE REVIEW

File: static/styles/main.css [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  background-color: rgba(255, 0, 0, 0.644);
+  border-radius: 0.75rem;
+ }
+
+ .menu .logout:active{
+  color: #ffffff;
+  background-color: rgb(181, 0, 0);
+  border-radius: 0.75rem;
+ }
+
```
Comment (TomerNewmanPrograms) [inline static/styles/main.css:170]: Should be here a newline

**Review by TomerNewmanPrograms — CHANGES_REQUESTED** [review]

File: static/styles/main.css (hunk 1 above)
Comment (OmerCS8) [reply static/styles/main.css]: Why? 170 is a blank line.

Comment (TomerNewmanPrograms) [reply static/styles/main.css]: Its ok now

**Review by TomerNewmanPrograms — DISMISSED** [review]

**Review by yulisuliman — APPROVED** [review]

**Review by MatanP12 — APPROVED** [review]
LGTM

**Review by AlonShakaroff — DISMISSED** [review]
LGTM

**Comment by natifridman** [comment]
@OmerCS8 Is there any reason why some of the CSS files are in the `static` folder in the root folder and some are under `posts/static`?

**Comment by OmerCS8** [comment]
> @OmerCS8 Is there any reason why some of the CSS files are in the `static` folder in the root folder and some are under `posts/static`?

Yes, the CSS file that in the `root/static` folder belongs to `base.html`, and therefore all of the apps are using it.

**Review by yulisuliman — DISMISSED** [review]

**Review by TomerNewmanPrograms — APPROVED** [review]

**Review by MatanP12 — APPROVED** [review]
LGTM

File: static/styles/main.css [hunk 2]
```diff
@@ -0,0 +1,171 @@
+/* ---------------------------------------------------------- GoogLe Fonts ---------------------------------------------------------- */
+@import url('https://fonts.googleapis.com/css2?family=Oleo+Script+Swash+Caps:wght@400;700&family=Poppins:ital,wght@0,300;0,500;1,600&display=swap');
+/* 
+font-family: 'Oleo Script Swash Caps', cursive;
+font-family: 'Poppins', sans-serif;
+ */
```
Comment (natifridman) [inline static/styles/main.css:6]: Please remove commented code

**Review by natifridman — CHANGES_REQUESTED** [review]

File: static/styles/main.css (hunk 2 above)
Comment (OmerCS8) [reply static/styles/main.css]: Done 👍

**Review by TomerNewmanPrograms — APPROVED** [review]
LGTm

**Review by yulisuliman — DISMISSED** [review]

**Review by AlonShakaroff — DISMISSED** [review]
LGTM

**Review by MatanP12 — DISMISSED** [review]
LGTM

**Review by TomerNewmanPrograms — APPROVED** [review]
LGTM

**Review by AlonShakaroff — APPROVED** [review]

**Review by yulisuliman — APPROVED** [review]

**Review by natifridman — APPROVED** [review]

**Comment by yulisuliman** [comment]
@OmerCS8  Please squash all commit into one

**Pull request merged**